docker-compose.yml

```diff
@@ -31,7 +31,8 @@ services:
             "credentials": {
                "CONSOLE_CLIENT_ID": "dashboard-local",
                "CONSOLE_CLIENT_SECRET": "notarealsecret",
-               "SESSION_KEY": "notarealsessionkey",
+               "CSRF_KEY": "notarealcsrfkey",
```

Comment:
these two will cause the server to not start because hex.Decode looks for `0-9a-f` (only hex) you could just make these both `0000` or `deadbeef` or some hex that wouldn't be used for both of them. then just put a yaml comment saying these aren't real

Reply:
Fixed, though I didn't add the comment as that part is JSON, and JSON (sadly) doesn't support comments.